scripts/legacy_expression_tool/README.md

```diff
@@ -204,3 +204,38 @@ python legacy_expression_tool.py \
     --translation <translation-file> \
     --commit
 ```
+
+### Output statistics
+
+When the script is executed, it prints some statistics about the matches found. This statistics can be printed filtered
+by service or subservice. By default, no statistics are printed. You can change it using the `--statistics` plus the
+aggregation mode. The possible values are `service` and `subservice`.
+
+```bash
+Found 2 legacy expressions in 2 documents
+                                                        _id
+service                                         service1 service2 All
+expression
+${@value*100/total}                                    1        1   2
+All                                                    1        1   2
+```
+
+### Known issues
+
+#### Execution with `expressionlanguage` set to `jexlall`
+
+When executing the script with `expressionlanguage` set to `jexlall`, the script will look for all the documents
+containing legacy expressions or the parameter `expressionLanguage`. This would change the number of documents found,
```

Comment:
What "containing legacy expression" mean? Should be detailed.

I understand that when `jexl` is used the statistics are accurate. If I'm right, then I'd suggest to explicitly say it here.

Reply:
Solved in https://github.com/telefonicaid/iotagent-node-lib/pull/1405/commits/acc754a5baf9076af420431aade4b6cd76e46ea1